lib/moon/design/carousel/carousel_control.ex

```diff
@@ -33,6 +33,7 @@ defmodule Moon.Design.Carousel.CarouselControl do
         "h-8 w-8 font-medium",
         "leading-[0] no-underline cursor-pointer absolute top-1/2 -translate-y-1/2",
         "origin-[top_center] z-5 disabled:opacity-60 disabled:cursor-not-allowed",
+        "rtl:hidden",
```

Comment:
In that case, indicators and select index variants are still broken. 
I would rather stick to the currently published version. What do you think?